src/solver/conv_winoRxS_f3x2.cpp

```diff
@@ -376,13 +376,13 @@ bool ConvBinWinogradRxSf3x2::IsApplicable(const ConvolutionContext& params) cons
 
 /// \todo Consider re-using code from RxS_f2x3.
 ConvSolution
-ConvBinWinogradRxSf3x2::GetSolution(const ConvolutionContext& params,
-                                    const PerformanceConfigConvBinWinogradRxSf3x2& config,
-                                    const bool disableConfigOverrideFromEnv) const
+ConvBinWinogradRxSf3x2::GetSolutionCTS(const ConvolutionContext& params,
+                                       const PerformanceConfigConvBinWinogradRxSf3x2& config) const
 {
     const PerformanceConfigConvBinWinogradRxSf3x2* pcfg = &config;
+
+    // Try to load config from environment variable
```

Comment:
[Style][Recommendation] Same as https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1433#discussion_r858770097

Reply:
Comment has been removed

Reply:
Thanks. Please hide resolved comments (I can't do that)